src/components/form/examples/row-layout-schema.ts

```diff
@@ -19,6 +19,15 @@ export const schema = {
                     description: 'Select the app language',
                     default: 'ua',
                     lime: {
+                        help: {
+                            value: '**Why do we need this information?** This enables our sales representatives to contact you personally. We will keep all your personal information safe!',
+                            readMoreLink: {
+                                href: 'https://www.lime-technologies.com/en/av-saas/',
+                                title: 'This link opens in a new tab',
+                                target: '_blank',
+                                text: 'Our Terms & Privacy Policies',
+                            },
+                        },
```

Comment:
we need another default placement for this layout type 👇 

<img width="427" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/0f69ee79-ea04-4aac-832e-700227281515">